targets/psa_switch/psa_switch.cpp

```diff
@@ -363,49 +486,25 @@ PsaSwitch::ingress_thread() {
     deparser->deparse(packet.get());
 
     auto &f_packet_path = phv->get_field("psa_egress_parser_input_metadata.packet_path");
-    const auto &f_ig_cos = phv->get_field("psa_ingress_output_metadata.class_of_service");
-    const auto ig_cos = f_ig_cos.get_uint();
-
-    // handling multicast
-    unsigned int mgid = 0u;
-    const auto &f_mgid = phv->get_field("psa_ingress_output_metadata.multicast_group");
-    mgid = f_mgid.get_uint();
 
+    unsigned int mgid = phv->get_field("psa_ingress_output_metadata.multicast_group").get_uint();
```

Comment:
same as above: `auto mgid = ...` for style consistency

Reply:
making this change in next commit